Artigo do Douglas Taquary. 30-03-17-core-data

douglastaquary opened the pull request.

douglastaquary committed:
Artigo do Douglas Taquary concluido. 30-03-17-core-data

rafaelnobrepd committed:
Revisão ortográfica

nobre84 reviewed (commented):
Fiz uma revisão ortográfica no seu PR, falta ajustar a data pro dia certo

douglastaquary committed:
Merge pull request #2 from nobre84/douglastaquary-30-03-17-intro-core-data-review-nobre84

Revisão ortográfica

douglastaquary committed:
Update 2017-03-30-persistencia-de-dados-usando-core-data.md

• Alterado data do post de `2016-03-15` para a `2017-03-30`.
• Correções ortográficas verificadas.

douglastaquary committed:
Update 2017-03-30-persistencia-de-dados-usando-core-data.md

nobre84 reviewed (approved):
👍 🚀
